pdr_backend/cli/arg_feed.py

```diff
@@ -124,12 +167,34 @@ def _unpack_feeds_str(feeds_str: str) -> List[ArgFeed]:
 
     exchange_str = feeds_str_split[0]
 
-    timeframe_str = feeds_str_split[-1]
+    threshold_str = feeds_str_split[-1]
     offset_end = None
 
+    if vb_str_ok(threshold_str):
```

Comment:
is there a possibility of multiple thresholds to be in place? If so, how is that managed?

Reply:
no, for each feed it can only be one of volume bar or tickbar or dollar bar